### PR TITLE
Autofix Rubocop Layout - Batch 5

### PR DESCRIPTION
**Pull request by nschonni**
Autofixed a bunch of the Layout rules, avoiding the ones deailing with operator spacing

### CODE REVIEW

File: test/json_gem/json_addition_test.rb [hunk 1]
```diff
@@ -13,7 +13,7 @@
   require 'json/add/bigdecimal'
   require 'json/add/ostruct'
 else
-  #Oj.add_to_json()
+  # Oj.add_to_json()
```
Comment (nschonni) [inline test/json_gem/json_addition_test.rb:16]:
```suggestion
```

File: test/test_fast.rb [hunk 2]
```diff
@@ -108,7 +108,7 @@ def test_float_exp
     json = %{12345.6789e7}
     Oj::Doc.open(json) do |doc|
       assert_equal(Float, doc.type)
-      #assert_equal(12345.6789e7, doc.fetch())
+      # assert_equal(12345.6789e7, doc.fetch())
```
Comment (nschonni) [inline test/test_fast.rb:111]:
```suggestion
```

File: test/test_fast.rb [hunk 3]
```diff
@@ -206,8 +207,7 @@ def test_move_relative
        ['/array/1/hash', 'h2/a', '/array/1/hash/h2/a'],
        ['/array/1', 'hash/h2/a/2', '/array/1/hash/h2/a/2'],
        ['/array/1/hash', '../string', '/array/1/string'],
-       ['/array/1/hash', '..', '/array/1'],
-      ].each do |start, path, where|
+       ['/array/1/hash', '..', '/array/1'],].each do |start, path, where|
```
Comment (nschonni) [inline test/test_fast.rb:210]: Probably need to manually adjust to prevent folding

File: test/test_fast.rb [hunk 4]
```diff
@@ -224,8 +224,7 @@ def test_type
        ['/array/1/string', String],
        ['/array/1/hash/h2/a', Array],
        ['/array/1/hash/../num', Integer],
-       ['/array/1/hash/../..', Array],
-      ].each do |path, type|
+       ['/array/1/hash/../..', Array],].each do |path, type|
```
Comment (nschonni) [inline test/test_fast.rb:227]: Probably need to manually adjust to prevent folding

File: test/test_fast.rb [hunk 5]
```diff
@@ -241,8 +240,7 @@ def test_local_key
        ['/array/1/hash/h2/a', 'a'],
        ['/array/1/hash/../num', 'num'],
        ['/array/1/hash/..', 1],
-       ['/array/1/hash/../..', 'array'],
-      ].each do |path, key|
+       ['/array/1/hash/../..', 'array'],].each do |path, key|
```
Comment (nschonni) [inline test/test_fast.rb:243]: Probably need to manually adjust to prevent folding

File: test/test_fast.rb [hunk 6]
```diff
@@ -263,8 +261,7 @@ def test_fetch_move
        ['/array/1/hash/../..', [{'num' => 3, 'string' => 'message', 'hash' => {'h2' => {'a' => [1, 2, 3]}}}]],
        ['/array/1', {'num' => 3, 'string' => 'message', 'hash' => {'h2' => {'a' => [1, 2, 3]}}}],
        ['/array', [{'num' => 3, 'string' => 'message', 'hash' => {'h2' => {'a' => [1, 2, 3]}}}]],
-       ['/', {'array' => [{'num' => 3, 'string' => 'message', 'hash' => {'h2' => {'a' => [1, 2, 3]}}}], 'boolean' => true}],
-      ].each do |path, val|
+       ['/', {'array' => [{'num' => 3, 'string' => 'message', 'hash' => {'h2' => {'a' => [1, 2, 3]}}}], 'boolean' => true}],].each do |path, val|
```
Comment (nschonni) [inline test/test_fast.rb:264]: Probably need to manually adjust to prevent folding

File: test/test_fast.rb [hunk 7]
```diff
@@ -283,8 +280,7 @@ def test_fetch_path
        ['/array', [{'num' => 3, 'string' => 'message', 'hash' => {'h2' => {'a' => [1, 2, 3]}}}]],
        ['/', {'array' => [{'num' => 3, 'string' => 'message', 'hash' => {'h2' => {'a' => [1, 2, 3]}}}], 'boolean' => true}],
        ['/nothing', nil],
-       ['/array/10', nil],
-      ].each do |path, val|
+       ['/array/10', nil],].each do |path, val|
```
Comment (nschonni) [inline test/test_fast.rb:283]: Probably need to manually adjust to prevent folding

File: test/test_fast.rb [hunk 8]
```diff
@@ -307,8 +303,7 @@ def test_move_fetch_path
     Oj::Doc.open(@json1) do |doc|
       [['/array/1', 'num', 3],
        ['/array/1', 'string', 'message'],
-       ['/array/1/hash', 'h2/a', [1, 2, 3]],
-      ].each do |path, fetch_path, val|
+       ['/array/1/hash', 'h2/a', [1, 2, 3]],].each do |path, fetch_path, val|
```
Comment (nschonni) [inline test/test_fast.rb:306]: Probably need to manually adjust to prevent folding

File: test/test_fast.rb [hunk 9]
```diff
@@ -322,8 +317,7 @@ def test_exists
        ['/array/1/hash', true],
        ['/array/1/dash', false],
        ['/array/3', false],
-       ['/nothing', false],
-      ].each do |path, val|
+       ['/nothing', false],].each do |path, val|
```
Comment (nschonni) [inline test/test_fast.rb:320]: Probably need to manually adjust to prevent folding

**Review by nschonni — COMMENTED** [review]
Few unrelated suggestions for what looks like dead/duplicate code. Also a few notes on some lines I figure you'll want reverted

File: test/test_fast.rb (hunk 3 above)
Comment (ohler55) [reply test/test_fast.rb:209]: Better the original way.

File: test/test_object.rb [hunk 10]
```diff
@@ -538,7 +534,7 @@ def test_ruby_time_utc
     # mismatch so instead the seconds, nsecs, and gmt_offset are checked
     # separately along with utc.
     json = Oj.dump(t, :mode => :object, :time_format => :ruby)
-    #puts json
+    # puts json
```
Comment (nschonni) [inline test/test_object.rb:537]: There are a bunch of these debug commented out chunks, but I left them as-is

File: test/json_gem/json_generic_object_test.rb [hunk 11]
```diff
@@ -54,8 +54,9 @@ def test_parse_json
   end
 
   def test_from_hash
-    result  = JSON::GenericObject.from_hash(
-      :foo => { :bar => { :baz => true }, :quux => [ { :foobar => true } ] })
+    result = JSON::GenericObject.from_hash(
+      :foo => { :bar => { :baz => true }, :quux => [ { :foobar => true } ] }
```
Comment (ohler55) [inline test/json_gem/json_generic_object_test.rb:58]: I thought we were going with tight brackets (no space).

**Comment by ohler55** [comment]
Odd that in some cases there are spaces near the brackets and in some cases not.

**Comment by ohler55** [comment]
No longer relevant.

**Pull request closed without merging**